Makefile

```diff
@@ -148,3 +148,12 @@ web/frontend/assets/images:
 .PHONY: helm-lint
 helm-lint:
 	docker run --rm -ti --name trento-chart-test -w /workdir -v $(shell pwd):/workdir quay.io/helmpack/chart-testing:v3.4.0 ct lint
+
+.PHONY: install-wwhrd
+install-wwhrd:
+	which wwhrd || go install github.com/frapposelli/wwhrd@latest
```

Comment:
I don't know that much about golang dependencies, but I wonder if there is some option to define this dependency as a `development` dependency

Reply:
I am not aware of such option. Did some research and could not find it either.

Reply:
I would @fabriziosestito since he is the lord of that pattern but I think we can actually declare `wwhrd` in the `tools/tools.go` file having it fetched inside deps